### PR TITLE
Also copy over subtask assignee on duplicate

### LINKED ISSUE #4469: Recurrent task does not copy over subtask assignees
### Actual behaviour

When a recurrent task is created it copies over the subtasks but fails to copy over subtask assignees

### Expected behaviour

Subtask assignees should also be copied

### Steps to reproduce

Just create a recurrent task with a subtask and assign it to someone, then force the recurrence to occur and look at the copied task.

### Configuration

- Kanboard version: dev-version from github on 2020-04-07
- Database type and version: sqlite3
- PHP version: 7.3
- OS: Ubuntu 18.4 LTS
- Browser: firefox/chrome/...

### PR DESCRIPTION
**Pull request by liedekef**
This resolves #4469

### CODE REVIEW

**Comment by BlueTeck** [comment]
The unit test is failing:

https://github.com/kanboard/kanboard/blob/master/tests/units/Model/SubtaskModelTest.php#L156

**Comment by liedekef** [comment]
Yes, it seems the unit test is checking that user_id is empty. So should I change my pull request to adapt the unit test too?

File: tests/units/Model/SubtaskModelTest.php [hunk 1]
```diff
@@ -152,9 +152,6 @@ public function testDuplicate()
         $this->assertEquals(0, $subtasks[0]['status']);
         $this->assertEquals(0, $subtasks[1]['status']);
 
-        $this->assertEquals(0, $subtasks[0]['user_id']);
-        $this->assertEquals(0, $subtasks[1]['user_id']);
-
```
Comment (fguillot) [inline tests/units/Model/SubtaskModelTest.php:157]: The unit test should validate that the assignee is correctly copied.

Comment (liedekef) [reply tests/units/Model/SubtaskModelTest.php:157]: Agreed, I'll adapt the unit test again

**Pull request merged**